packages/perspective-jupyterlab/src/ts/plugin.ts

```diff
@@ -19,16 +19,19 @@ import {PerspectiveView} from "./view";
 
 import {PERSPECTIVE_VERSION} from "./version";
 
+const EXTENSION_ID = "@finos/perspective-jupyterlab";
+
 /**
  * PerspectiveJupyterPlugin Defines the Jupyterlab plugin, and registers `PerspectiveModel` and `PerspectiveView`
  * to be called on initialization.
  */
 export const PerspectiveJupyterPlugin: IPlugin<Application<Widget>, void> = {
-    id: "@finos/perspective-jupyterlab",
+    id: EXTENSION_ID,
+    // @ts-ignore
```

Comment:
For the record, I, personally, don't care about the use of `@ts-ignore`.  But, I can't pass up on the opportunity to point out, only mostly satirically, that there's a great TypeScript option for disabling type errors called _JavaScript_ that would work wonders here.

Reply:
we're bound somewhat by the examples published by JupyterLab, the typescript extension is really the only maintained one it seems.

Reply:
Sounds like what you're saying is, if I were to de-TypeScript this module, you wouldn't object?

Reply:
Nope, `ts-ignore` is the final straw for me. It's working perfectly and yet due to the craziness of widgets and jlab 3 it will never compile. If you de-typescript i think you can also delete the perspective typings.